parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -187,6 +187,54 @@ write_files:
     {{WrapAsVariable "systemdBPFMount"}}
 {{end}}
 
+{{if NeedsContainerd}}
+- path: /etc/containerd/config.toml
+  permissions: "0644"
+  owner: root
+  content: |
+    subreaper = false
+    oom_score = 0
+    [plugins.cri]
+    sandbox_image = "{{GetPodInfraContainerSpec}}"
+    [plugins.cri.containerd.untrusted_workload_runtime]
+    runtime_type = "io.containerd.runtime.v1.linux"
+{{if IsKataContainerRuntime }}
+    runtime_engine = "/usr/bin/kata-runtime"
+{{else}}
+    runtime_engine = "/usr/local/sbin/runc"
+{{end}}
+    [plugins.cri.containerd.default_runtime]
+    runtime_type = "io.containerd.runtime.v1.linux"
+    runtime_engine = "/usr/local/sbin/runc"
```

Comment:
I think this needs the same if/else to work with kata

Reply:
or we could cut kata support if it's not tested

Reply:
I was hesitant to cut support for kata as I wasn't sure it it was in use. I've left it in for now. I propose we make that choice and remove it completely as a separate change.

Reply:
sounds good